packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -321,6 +323,14 @@ export class CapabilityService {
     targetPlanId: string,
     useFirestoreProductConfigs = false
   ): Promise<SubscriptionChangeEligibility> {
+    const emptyEligibilityResult: {
+      subscriptionEligibilityResult: SubscriptionEligibilityResult;
+      eligibleSourcePlan?: AbbrevPlan;
+    } = {
+      subscriptionEligibilityResult: SubscriptionEligibilityResult.INVALID,
+    };
+    const contentfulEnabled = config.getProperties().contentful.enabled;
```

Comment:
Instead of importing `config`, could you rather follow the pattern followed by other Class implementation and fetch it from the `Container`? A good example is the `AccountDeleteManager` ([Code here](https://github.com/mozilla/fxa/blob/52142f00f3851a3636e272790204afa6ede32b5d/packages/fxa-auth-server/lib/account-delete.ts#L129))